src/hera/retry.py

```diff
@@ -17,8 +19,9 @@ class Retry(BaseModel):
     """
 
     duration: Optional[int]
-    max_duration: Optional[int]
```

Comment:
Any chance you're open to adjusting the docstring to reflect the available fields? 🙂